tests/Main.m

```diff
@@ -1,6 +1,7 @@
 classdef Main < ...
     TestConfig & ...
     TestConnection & ...
+    TestTableDecleration & ...
```

Comment:
@Synicix The convention that has emerged here is mostly related to calling the test class the similar to the DataJoint class and also there is a typo with your naming here. Since it is about testing `dj.internal.Declare` (which is about how to compose SQL for table declarations), I better name might be simply `TestDeclare`.